src/components/Home.jsx

```diff
@@ -5,29 +5,59 @@ import Upload from './Upload';
 
 const Home = ({ setActors, actors, currentScript, setCurrentScript }) => {
   const [titles, setTitles] = useState([]);
+  const [deleteError, setDeleteError] = useState('');
+  const [addError, setAddError] = useState('');
   const newActorInput = useRef('');
 
   // submit of actor name handler
   const newActorHandler = (event) => {
     event.preventDefault();
     const newActor = { name: newActorInput.current.value };
 
-    // Add the new actor to the database
+    // Add the new actor to the database, returning the new actor object
     fetch('/actors', {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
       },
       body: JSON.stringify(newActor),
     })
-      .then(() => {
-        // set actors in state to be our new actors
-        setActors([...actors, newActor]);
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw new Error('Error when adding the actor');
+        }
+      })
+      .then((actor) => {
+        setActors([...actors, actor]);
         // reset input value to empty string to clear out old input
         newActorInput.current.value = '';
+        setAddError('');
+      })
+      .catch((error) => {
+        setAddError(error.message);
+      });
+  };
+
+  const deleteActor = (event) => {
+    const actor = event.target.value.split(',');
```

Comment:
`event.target.value.split(',');` < this is a bit confusing on its own and should probably have a comment describing what data is contained there that we're splitting. This is rather unusual to have multiple bits of data in a button value.  
Per our discussion, this should probably just be the id since that's all that is needed to do a delete.